benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/internal_data/hammerdb_vm_template.yaml

```diff
@@ -74,14 +74,14 @@ spec:
       kind: vm
       client_vm:
         dedicatedcpuplacement: false
-        sockets: 1
-        cores: {{ cores }}
+        sockets: {{ vm_sockets }}
+        cores: {{ vm_cores }}
         threads: 1
         image: quay.io/cloud-bulldozer/{{ db_image }}
         limits:
-          memory: {{ limits_memory_GB }}Gi
+          memory: {{ vm_limits_memory_GB }}Gi
         requests:
-          memory: 10Mi
+          memory: {{ vm_requests_memory_GB }}Gi
```

Comment:
May want to add cpu requests for the VM as well otherwise you are only guaranteed 1/10th of a cpu worth of shares. If you are making sockets and cores both configurable, the total cpu requests would be sockets * cores.

Reply:
Yes, I configure the cpu and socket accordingly